docs/readthedocs/usersguide.rst

```diff
@@ -183,7 +183,7 @@ Requirements
 
 - The external simulation should be implemented in its own project (repository).
 - The project should include the *shadowJar* gradle plugin (``id "com.github.johnrengelman.shadow" version "x.y.z"``).
-- A class (called *main class* here) needs to extend ``edu.ie3.simona.api.schedule.ExtSimulation`` and thus implement ``List<Long> doActivity(long tick)``, which is called when time step ``tick`` is triggered. ``doActivity`` must return a list of subsequent new ticks that the sub simulation should be scheduled at.
+- A class (called *main class* here) needs to extend ``edu.ie3.simona.api.schedule.ExtSimulation`` and thus implement the two functions ``List<Long> initialize()´´ and ``List<Long> doActivity(long tick)``. The function ``initialize´´ is called when the external simulation needs to be initialized whereas the function ``doActivity´´ is called when time step ``tick`` is triggered. ``initialize´´ and ``doActivity`` must return a list of subsequent new ticks that the sub simulation should be scheduled at.
```

Comment:
I'd call them `methods` instead of `functions` (check [here](https://stackoverflow.com/questions/155609/whats-the-difference-between-a-method-and-a-function) for the difference between the two)